build.gradle

```diff
@@ -39,14 +39,15 @@ processResources {
 }
 
 dependencies {
-    compile("org.springframework.boot:spring-boot-starter-data-mongodb")
-    compile("org.springframework.boot:spring-boot-starter-web")
-	testCompile("org.mockito:mockito-core")
-	testCompile("org.springframework.boot:spring-boot-starter-test")
+    implementation("org.springframework.boot:spring-boot-starter-data-mongodb")
+    implementation("org.springframework.boot:spring-boot-starter-web")
+    testImplementation("org.mockito:mockito-core")
+    testImplementation("org.springframework.boot:spring-boot-starter-test")
+    testImplementation 'junit:junit:4.12'
 }
 
 springBoot {
-    mainClassName = "org.kpmp.Application"
+    mainClass = "org.kpmp.Application"
```

Comment:
Ditto for this instances as well regarding downstream issues

Reply:
Ditto above.